hometask-2/scripts/create_and_populate_users_table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE users (
```

Comment:
As future reading, you can have a look at database schema migrations as strategy to maintain your DB schema;
Sequelize has the infrastructure to do it, but I'm not a big fan because it makes it coupled to Sequelize and I prefer to have it in raw SQL (as you have).
https://sequelize.org/docs/v6/other-topics/migrations/